plate_app/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from .models import Plate, Ingredient, Nutrient, Measurement
+
+class PlateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Plate
+        fields = ['id','created_at']
+
+class IngredientSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ingredient
+        fields = ['id','name','measurement_id','nutrients_id']
```

Comment:
remember to use `fields = ["__all__"]` whenever possible to avoid writing each individual field